FirebaseCore/Extension/FIRLogger.h

```diff
@@ -127,26 +127,26 @@ extern void FIRLogDebug(FIRLoggerService service, NSString *messageCode, NSStrin
  * not log any messages with a level higher than FirebaseLoggerLevelNotice to avoid log spamming.
  * This function is intended to be used by Swift clients that do not support variadic parameters.
  *
- * @param service The service name of type `FirebaseLoggerService`.
+ * @param category The service name of type `FirebaseLoggerService`.
  * @param messageCode The mesage code. starting with "I-" which means iOS, followed by a capitalized
  * three-character service identifier and a six digit integer message ID that is unique within the
  * service. An example of the message code is @"I-COR000001".
  * @param message The message string.
  */
-extern void FIRLogDebugSwift(FIRLoggerService service, NSString *messageCode, NSString *message);
+extern void FIRLogDebugSwift(NSString *category, NSString *messageCode, NSString *message);
 
 /**
  * Logs a warning message to the Xcode console and the device log. If running from AppStore, will
  * not log any messages with a level higher than FirebaseLoggerLevelNotice to avoid log spamming.
  * This function is intended to be used by Swift clients that do not support variadic parameters.
  *
- * @param service The service name of type `FirebaseLoggerService`.
+ * @param category The service name of type `FirebaseLoggerService`.
  * @param messageCode The mesage code. starting with "I-" which means iOS, followed by a capitalized
  * three-character service identifier and a six digit integer message ID that is unique within the
  * service. An example of the message code is @"I-COR000001".
  * @param message The message string.
  */
-extern void FIRLogWarningSwift(FIRLoggerService service, NSString *messageCode, NSString *message);
+extern void FIRLogWarningSwift(NSString *category, NSString *messageCode, NSString *message);
```

Comment:
I think we should consider cleaning these up now. I'm not actually sure how'd they work. They're `extern` but I don't see an implementation in the codebase but I might be missing something.